packages/auth/migrations/20241125233415_create_tenants_table.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('tenants', function (table) {
+    table.uuid('id').notNullable().primary()
+    table.string('idpConsentUrl').notNullable()
+    table.string('idpSecret').notNullable()
+
+    table.timestamp('createdAt').defaultTo(knex.fn.now())
+    table.timestamp('updatedAt').defaultTo(knex.fn.now())
```

Comment:
Are these generated in this table, or passed over from backend?
Should we also include `deletedAt`?

Reply:
I would say that yes, this createdAt and updatedAt needs to be generated in this table since it is different entity. Bot somehow, still connected with the one in backend. with createdAt we get info when was the entity created in that database, not when it was created in the backed database. What do you think ?